.github/workflows/repo-request.yml

```diff
@@ -37,20 +61,43 @@ jobs:
           if [[ "$REPO" == -* ]]; then exit 1; fi
           # shall not end with minus
           if [[ "$REPO" == *- ]]; then exit 1; fi
+      - name: Get Attrbutes
+        id: get-attributes
+        env:
+          JSON: ${{ needs.provide-json.outputs.json }}
+        run: |
+          echo "visibility=$(echo $JSON | jq -r '.Visibility')" >> "$GITHUB_OUTPUT"
+          echo "language=$(echo $JSON | jq -r '.Language')" >> "$GITHUB_OUTPUT"
       - name: Get Token
         id: get-workflow-token
         uses: peter-murray/workflow-application-token-action@v2
         with:
           application_id: ${{ vars.GH_APP_ID }}
           application_private_key: ${{ secrets.GH_APP_KEY }}
           organization: ${{ vars.ORGANIZATION }}
+      - name: Add files for initial commit
+        env:
+          LANGUAGE: ${{ steps.get-attributes.outputs.language }}
+        run: |
+          cp -r ./templates/${LANGUAGE}/* .
+          git add .
+      - name: Create commit
+        env:
+          CI_COMMIT_USER: "xpirit-training-bot[bot]"
+          CI_COMMIT_USER_MAIL: 139118093+xpirit-training-bot[bot]@users.noreply.github.com
+          CI_COMMIT_MESSAGE: "chore(org): Initial commit"
```

Comment:
same here -> into repo variable.

Reply:
mail could be a repo variable, message is different :)

Reply:
done